org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/completion/LSContentAssistProcessor.java

```diff
@@ -65,10 +65,10 @@ public class LSContentAssistProcessor implements IContentAssistProcessor {
 	private IDocument currentDocument;
 	private String errorMessage;
 	private final boolean errorAsCompletionItem;
-	private CompletableFuture<List<@NonNull LanguageServer>> completionLanguageServersFuture;
+	private CompletableFuture<@NonNull List<@NonNull Void>> completionLanguageServersFuture;
```

Comment:
`@NonNull Void` seems strange as the only value accepted by a reference of type `Void` is null.

Reply:
Yes, it is weird but without it the JDT null analysis creates markers, so I think it is better like this, to avoid the marker.

Thanks for the review